src/hera/workflows/task.py

```diff
@@ -47,60 +47,90 @@ class Task(
     ParameterMixin,
     ItemMixin,
 ):
-    r"""Task is used to run a given template within a DAG. Must be instantiated under a DAG context.
-
-    ## Dependencies
-    Any Tasks without a dependency defined will start immediately.
-
-    Dependencies between Tasks can be described using the convenience syntax `>>`, for example:
-        A = Task(...)
-        B = Task(...)
-        A >> B
-    describes the relationships:
-    * "A has no dependencies (so starts immediately)
-    * "B depends on A".
-    As a diagram:
-    A
-    |
-    B
-
-    `A >> B` is equivalent to `A.next(B)`.
-
-    ## Lists of Tasks
-    A list of Tasks used with the rshift syntax describes an "AND" dependency between the single Task on the left of
-    `>>` and the list Tasks to the right of `>>` (or vice versa). A list of Tasks on both sides of `>>` is not supported.
-    For example:
-        A = Task(...)
-        B = Task(...)
-        C = Task(...)
-        D = Task(...)
-        A >> [B, C] >> D
-    describes the relationships:
-    * "A has no dependencies
-    * "B AND C depend on A"
-    * "D depends on B AND C"
-    As a diagram:
-      A
-     / \
-    B   C
-     \ /
-      D
-
-    Dependencies can be described over multiple statements:
-        A = Task(...)
-        B = Task(...)
-        C = Task(...)
-        D = Task(...)
-        A >> [C, D]
-        B >> [C, D]
-    describes the relationships:
-    * "A and B have no dependencies
-    * "C depends on A AND B"
-    * "D depends on A AND B"
-    As a diagram:
-    A   B
-    | X |
-    C   D
+    """Task is used to run a given template within a DAG. Must be instantiated under a DAG context.
+
+## Dependencies
+
+Any `Tasks` without a dependency defined will start immediately.
+
+Dependencies between Tasks can be described using the convenience syntax `>>`, for example:
+
+```py
+    A = Task(...)
+    B = Task(...)
+    A >> B
+```
+
+describes the relationships:
+
+* "A has no dependencies (so starts immediately)
+* "B depends on A".
+
+As a diagram:
+
+```
+A
+|
+B
+```
```

Comment:
This de-indenting is to fix the formatting in https://hera.readthedocs.io/en/stable/api/workflows/hera/#hera.workflows.Task